tuned_lens/scripts/downstream.py

```diff
@@ -21,10 +24,33 @@
 print "{}":\n\n"""
 
 
+@dataclass
+class CliArgs(MainCliArgs):
+    """Type hinting for CLI args."""
+
+    lens: Optional[Path] = field(nargs="?")
```

Comment:
should `lens` be required? it wasn't marked as required previously but the code below seems to expect it will have a value. same question for the other scripts.

Reply:
Yes it should be required going forward.